src/Castle.Windsor.Tests/Categories/UnitTestAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright 2004-2017 Castle Project - http://www.castleproject.org/
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace CastleTests.Categories
+{
+	using NUnit.Framework;
+
+	public class UnitTestAttribute : CategoryAttribute
+	{
+		public UnitTestAttribute() : base("Unit")
```

Comment:
Can you explain why we need 3 category attributes, I've done something similar before and don't understand why you'd need a `UnitTestAttribute`.